cfgov/jinja2/v1/_includes/organisms/post-preview.html

```diff
@@ -145,8 +145,8 @@
                 <time class="calendar-icon"
                       data-month="{{ event.start_dt | date('%b') }}"
                       data-day="{{ event.start_dt | date('%-d') }}"
-                      datetime="{{ event.start_dt | date('%c') }}">
-                    <span class="u-visually-hidden">{{ event.start_dt | date('%b %-d, %Y') }}</span>
+                      datetime="{{ event.start_dt | date('%c') }}"
+                      aria-label="{{ event.start_dt | date('%b %-d, %Y') }}">
```

Comment:
In this case, It's just a visual flourish that's repeating the main date slug that preceded it, so let's hide this one from screen readers. Since we changed the layout of these previews to put the image on the right, it makes less sense to have this at all anymore, but removing it is a discussion for another day.

```suggestion
                      aria-label="{{ event.start_dt | date('%b %-d, %Y') }}">
                      aria-hidden="true">
```

Reply:
This would have only `aria-hidden` and not `aria-label` then.

Reply:
Yes, that's right.

Reply:
Fixed

Reply:
Oh, I see, I messed up my suggestion 😆